packages/forklift-console-plugin/src/modules/Plans/views/list/components/VMsProgressCell.tsx

```diff
@@ -53,19 +55,37 @@ export const VMsProgressCell: React.FC<CellProps> = ({ data }) => {
   const progressVariant = getPlanProgressVariant(phase);
 
   return (
-    <Link to={`${planURL}/vms`}>
-      <div className="forklift-table__status-cell-progress">
-        <Progress
-          width={100}
-          label={t('{{success}} of {{total}} VMs migrated', counters)}
-          valueText={t('{{success}} of {{total}} VMs migrated', counters)}
-          value={counters?.total > 0 ? (100 * counters?.success) / counters?.total : 0}
-          title={t(phaseLabel)}
-          size={ProgressSize.sm}
-          measureLocation={ProgressMeasureLocation.top}
-          variant={progressVariant}
-        />
-      </div>
-    </Link>
+    <div className="forklift-table__status-cell-progress">
+      <Progress
+        width={100}
+        label={
+          <Link to={`${planURL}/vms`}>{t('{{success}} of {{total}} VMs migrated', counters)}</Link>
+        }
+        valueText={t('{{success}} of {{total}} VMs migrated', counters)}
+        value={counters?.total > 0 ? (100 * counters?.success) / counters?.total : 0}
+        title={
+          <Popover
+            headerContent={<div>{t('Status details')}</div>}
+            bodyContent={
+              <div>
```

Comment:
Looks great, 
Can you add a line for the total of all vms in this plan ?
"[VMs in the plan] virtual machines planned for migration."
Can you talk to @RichardHoch about the exact wards to use, maybe "[Total VMs failed] virtual machined failed" e.g. "3 virtual machines failed"?

Reply:
I didn't add the "total num of VMs" detail to the status popover because there is already a dedicated "`virtual machines`" column for that: 
[/home/sgratch/go/src/github/kubev2v/forklift-console-plugin/packages/forklift-console-plugin/src/modules/Plans/views/list/components/VMsCell.tsx](https://github.com/kubev2v/forklift-console-plugin/blob/main/packages/forklift-console-plugin/src/modules/Plans/views/list/components/VMsCell.tsx)

Reply:
We can easily add it to the popover as well though. Just to display the "complete picture" regarding the plan's VMs

Reply:
> Can you add a line for the total of all vms in this plan ?

Done (even though not sure it's needed 

> Can you talk to @RichardHoch about the exact wards to use

Waiting for Richard's review on the rephrasing.